config/default.yml

```diff
@@ -1,5 +1,5 @@
 server_port: 9001
-service_map_backend: https://api.hel.fi/servicemap/v1
+service_map_backend: http://localhost:8000/v2
```

Comment:
You are chaning to localhost here and in the next commit it is changed back to api.hel.fi